charts/govuk-apps-conf/templates/configmap.yaml

```diff
@@ -13,6 +13,7 @@ data:
   GOVUK_ASSET_ROOT: https://assets.{{ .Values.publishingServiceDomainSuffix }}
   GOVUK_ENVIRONMENT: {{ .Values.govukEnvironment }}
   GOVUK_ENVIRONMENT_NAME: {{ .Values.govukEnvironment }}
+  GOVUK_PROMETHEUS_EXPORTER: "{{ .Values.govukPrometheusExporter }}"
```

Comment:
No need to put this in the configmap, because it's not really a global setting. It'd make more sense in the app charts (`govuk-rails-app`, `asset-manager` and any others that have been split off). I would just set it right in the Deployment template, in the `env` section.

Reply:
Actually, sorry, scratch that - it's fine either way. I would just set it to true though rather than adding a value that we're never going to use.

Reply:
make sense, have changed to be directly in the configmap.